docs/usage-as-end-user.md

```diff
@@ -62,6 +65,8 @@ The `networks.internal` section is optional and can describe a CIDR for a subnet
 
 The `networks.cloudNAT.minPortsPerVM` is optional and is used to define the [minimum number of ports allocated to a VM for the CloudNAT](https://cloud.google.com/nat/docs/overview#number_of_nat_ports_and_connections)
 
+The `networks.cloudNAT.natIPNames` is optional and is used to specify the names of the manual ip addresses which should be used by the nat gateway
```

Comment:
Same here, this document lacks the example for IPs instead of names.

Reply:
See comment above.